learning_assistant/serializers.py

```diff
@@ -0,0 +1,40 @@
+"""
+Serializers for the learning-assistant API.
+"""
+from rest_framework import serializers
+
+
+class MessageSerializer(serializers.Serializer):
+    """
+    Serializer for a message.
+    """
+
+    role = serializers.CharField(required=True)
+    content = serializers.CharField(required=True)
+
+    def validate_role(self, value):
+        """
+        Validate that role is one of two acceptable values.
+        """
+        valid_roles = ['user', 'assistant']
+        if value not in valid_roles:
+            raise serializers.ValidationError('Must be valid role.')
+        return value
+
+    def update(self, instance, validated_data):
```

Comment:
I'm not sure if this is the right approach, but in order to use the BaseSerializer I need to implement both `update` and `create`, even if they're not being used (otherwise pylint will complain)

Reply:
That's interesting. The DRF documentation has a serializer with just the serializer fields declared. What's the Pylint rule that's failing?

Reply:
Maybe it would make more sense to ignore the pylint errors for now, as none of the abstract classes are being used for the serializer right now.

Reply:
I think that's valid. I see that pattern in [registrar](https://github.com/openedx/registrar/blob/master/registrar/apps/grades/serializers.py#L7) as well.